hokusai/lib/common.py

```diff
@@ -78,10 +78,13 @@ def returncode(command, mask=()):
   return call(verbose(command, mask=mask), stderr=STDOUT, shell=True)
 
 def shout(command, print_output=False, mask=()):
-  if print_output:
-    return check_call(verbose(command, mask=mask), stderr=STDOUT, shell=True)
-  else:
-    return check_output(verbose(command, mask=mask), stderr=STDOUT, shell=True)
+  try:
+    if print_output:
+      return check_call(verbose(command, mask=mask), stderr=STDOUT, shell=True)
+    else:
+      return check_output(verbose(command, mask=mask), stderr=STDOUT, shell=True)
+  except CalledProcessError as e:
+    raise HokusaiError(str(e), return_code=e.returncode, mask=mask)
```

Comment:
Passing the string representation of the [CalledProcessError](https://docs.python.org/2/library/subprocess.html#subprocess.CalledProcessError) rather than its output attribute as this may be `None` - also we want to ensure a string rather than unicode when it is caught and its message attribute is printed in `command.py`

Reply:
Actually seeing some perhaps unexpected changes to hokusai's behavior due to these changes... This introduces a change that the `CalledProcessError`'s returncode is passed through to `HokusaiError` - this effectively changes hokusai's behavior in that it will return the returncode of the error's subcommand rather than `1` - I was surprised we don't do this already - we are currently catching all CalledPRocessErrors separately [here](https://github.com/artsy/hokusai/blob/master/hokusai/lib/command.py#L29) and returning `1`.  Perhaps this exception handling needs to be refactored...